index.html

```diff
@@ -4975,6 +4975,634 @@ <h2 id="AudioWorkerParamDescriptor">
+                  <code>"name"</code>.
+                </dd>
+              </dl>
+            </dd>
+            <dt>
+              void postMessage()
+            </dt>
+            <dd>
+              <p>
+                It may be called to send a message to the
+                <a><code>AudioWorkletNode</code></a>, via the algorithm defined
+                by the Worker specification.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  any message
+                </dt>
+                <dd></dd>
+                <dt>
+                  optional sequence&lt;Transferable&gt; transfer
+                </dt>
+                <dd></dd>
+              </dl>
+            </dd>
+          </dl>
+          <section>
+            <h2 id="AudioParamDescriptor">
+              <dfn>AudioParamDescriptor</dfn>
+            </h2>
+            <p>
+              The <code>AudioParamDescriptor</code> dictionary is used to
+              specify properties for an <a><code>AudioParam</code></a> object
+              that is used in an <a><code>AudioWorkletNode</code></a>.
+            </p>
+            <dl title="dictionary AudioParamDescriptor" class="idl">
+              <dt>
+                DOMString name
+              </dt>
+              <dd>
+                Represents the name of a parameter. An NotSupportedError
+                exception MUST be thrown when a duplicated name is found when
+                registering the class definition.
+              </dd>
+              <dt>
+                float defaultValue = 0
+              </dt>
+              <dd>
+                Represents the default value of the parameter. If this value is
+                out of the range of float data type or the range defined by
+                <code>minValue</code> and <code>maxValue</code>, an
+                NotSupportedError exception MUST be thrown.
+              </dd>
+              <dt>
+                float minValue
+              </dt>
+              <dd>
+                Represents the minimum value. An NotSupportedError exception
+                MUST be thrown if this value is out of range of float data type
+                or it is greater than <code>maxValue</code>.
+              </dd>
+              <dt>
+                float maxValue
+              </dt>
+              <dd>
+                Represents the maximum value. An NotSupportedError exception
+                MUST be thrown if this value is out of range of float data type
+                or it is smaller than <code>minValue</code>.
+              </dd>
+            </dl>
+          </section>
+          <section>
+            <h2 id="AudioContextInfo">
+              <dfn>AudioContextInfo</dfn>
+            </h2>
+            <p>
+              The <code>AudioContextInfo</code> dictionary contains the status
+              of the associated <a><code>AudioContext</code></a>.
```

Comment:
I think it is clearer to say that an `AudioContextInfo` provides an `AudioWorkletGlobalScope` with a *view* of an `AudioContext`. There is more to this object than status.